frontend/src/metabase/home/homepage/components/HomeContent/HomeContent.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { isSyncCompleted } from "metabase/lib/syncing";
+import { Database } from "metabase-types/api";
+import XraySection from "../../containers/XraySection";
+
+export interface HomeContentProps {
+  databases: Database[];
+}
+
+const HomeContent = ({ databases }: HomeContentProps): JSX.Element | null => {
```

Comment:
This was moved to an individual component to not block rendering of `HomeLayout` until the content loads.